pyproject.toml

```diff
@@ -58,11 +58,6 @@ line_length = 100
 
 [tool.pylint.messages_control]
 disable = [
-        # pylint sometimes complains about black's formatting; we go with black
```

Comment:
It seems like these configs don't do anything in pylint any more; pylint itself told me to remove them, and removing them doesn't seem to have broken anything else.

Reply:
ha! nice. This is a good note, some context on the future for linting in this repo, we'll be switching to ruff https://beta.ruff.rs/docs/ (like were using in vunnel). It's fast and more feature full.